modules/core/src/passes/shadow-pass.js

```diff
@@ -52,30 +51,25 @@ export default class ShadowPass extends LayersPass {
         const width = viewport.width * this.props.pixelRatio;
         const height = viewport.height * this.props.pixelRatio;
         if (width !== target.width || height !== target.height) {
-          // this.shadowMap.resize({width, height});
           target.resize({width, height});
         }
 
         super.render(Object.assign(params, {outputBuffer: target}));
-        // this.target.generateMipmap();
       }
     );
   }
 
-  shouldDrawLayer(layer) {
-    return layer.props.castShadow;
-  }
-
   getModuleParameters(layer, effects, effectProps) {
     const moduleParameters = Object.assign(Object.create(layer.props), {
       viewport: layer.context.viewport,
-      pickingActive: 1,
+      pickingActive: 0,
```

Comment:
`pickingActive` was set to `1` to avoid calculating lighting during the shadow pass. Is this behavior preserved?

Reply:
yeah, the behavior is preserved cause there is no lighting effect passed to shadow pass rendering, the pickingActive need to 0 otherwise the picking filtering will corrupt the shadow map.